about.html

```diff
@@ -190,10 +192,20 @@ <h5 class="white-text">Contact Us</h5>
 
 
   <!--  Scripts-->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/tiny-slider/2.8.7/min/tiny-slider.js"></script>
   <script>
-    const mySiema = new Siema({loop: true, duration: 400})
-    document.querySelector('.prev-photo').addEventListener('click', () => mySiema.prev())
-    document.querySelector('.next-photo').addEventListener('click', () => mySiema.next())
+    const slider = tns({
```

Comment:
I'm not sure of our browser support but this should maybe be a `var` just in case